packages/curve-ui-kit/src/widgets/Footer/Description.tsx

```diff
@@ -0,0 +1,57 @@
+import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import Link from '@mui/material/Link'
+import Typography from '@mui/material/Typography'
+
+import { LogoImg } from 'ui/src/images'
+
+const LogoImageSrc = (LogoImg as unknown as { src: string }).src
+const Logo = styled('img')({
+  width: 48,
+  height: 48,
+  margin: 8,
+  alt: 'Curve',
+})
+
+export const Description = () => (
+  <Box
+    sx={{
+      display: 'flex',
+      flexDirection: 'column',
+      maxWidth: '330px',
+      mr: '1.5rem',
+    }}
+  >
+    <Link
```

Comment:
This looks like a duplicate of `<HeaderLogo appName="main" />` maybe we should [move this component](https://github.com/curvefi/curve-frontend/blob/ebd51245b139c68a1f4ad6703d27d998e749637d/packages/curve-common/src/widgets/Header/HeaderLogo.tsx#L18) to shared/ui too?

Reply:
I took the new header logo component as inspiration indeed, but had to make slight modifications as it's slightly different according to the Figma design. Up to @0xtutti if we actually want a carbon copy of said logo.